frontend/awx/resources/templates/TemplatesList.tsx

```diff
@@ -0,0 +1,93 @@
+import { useTranslation } from 'react-i18next';
+import {
+  IPageAction,
+  PageActionSelection,
+  PageActionType,
+  PageTable,
+  usePageNavigate,
+} from '../../../../framework';
+import { AwxRoute } from '../../AwxRoutes';
+import { useAwxView } from '../../useAwxView';
+import { JobTemplate } from '../../interfaces/JobTemplate';
+import { WorkflowJobTemplate } from '../../interfaces/WorkflowJobTemplate';
+import { usePersistentFilters } from '../../../common/PersistentFilters';
+import { useTemplateFilters } from './hooks/useTemplateFilters';
+import { useTemplateColumns } from './hooks/useTemplateColumns';
+import { useDeleteTemplates } from './hooks/useDeleteTemplates';
+import { useMemo } from 'react';
+import { PlusCircleIcon, TrashIcon } from '@patternfly/react-icons';
+import { ButtonVariant } from '@patternfly/react-core';
+import { useTemplateActions } from './hooks/useTemplateActions';
+
+export function TemplatesList(props: { url: string; projectId?: string }) {
+  const { t } = useTranslation();
+  const pageNavigate = usePageNavigate();
+  const toolbarFilters = useTemplateFilters();
+  const tableColumns = useTemplateColumns();
+  const view = useAwxView<JobTemplate | WorkflowJobTemplate>({
+    url: props.url,
+    queryParams: {
+      project__id: props.projectId ? props.projectId : '',
+      type: 'job_template,workflow_job_template',
+    },
+    toolbarFilters,
+    tableColumns,
+  });
+  usePersistentFilters('templates');
+  const deleteTemplates = useDeleteTemplates(view.unselectItemsAndRefresh);
+
+  const toolbarActions = useMemo<IPageAction<JobTemplate | WorkflowJobTemplate>[]>(
+    () => [
+      {
+        type: PageActionType.Dropdown,
+        variant: ButtonVariant.primary,
+        isPinned: true,
+        label: t('Create template'),
+        selection: PageActionSelection.None,
+        icon: PlusCircleIcon,
+        actions: [
+          {
+            type: PageActionType.Button,
+            selection: PageActionSelection.None,
+            label: t('Create job template'),
+            onClick: () => pageNavigate(AwxRoute.CreateJobTemplate),
```

Comment:
Use an `href` instead of `onClick` to render this as a normal link — `href: getPageUrl(AwxRoute.CreateJobTemplate)`

(we still have a lot of older examples in the code that use onClick from before we added the href support. See Organizations.tsx for an example of the href approach)

Reply:
@keithjgrant this is good to know, thanks for pointing that out!